### PR TITLE
ading-product-lpf-griml

### PR DESCRIPTION
**Pull request by open-science-data**
Ading a product matadata to LPF GRIML project

### CODE REVIEW

File: products/esa-griml-iml-fv1/collection.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
...SIDC-0714/versions/1) to clip the dataset to within 1 km of the ice margin. Each detected lake was then verified manually. The methodology is open-source and provided at https://github.com/GEUS-Glaciology-and-Climate/GrIML for full reproducibility. Please consult the dataset readme provided with this dataset for further information on the associated metadata. Acknowledgements: The inventory series of ice marginal lakes in Greenland has been produced as part of the European Space Agency (ESA) Living Planet Fellowship project "Examining GReenland’s Ice Marginal Lakes under a changing climate (GrIML)", which is a follow-on effort to the 2017 inventory of ice marginal lakes created under the European Space Agency (ESA) Climate Change Initiative (CCI) in Option 6 of the Glaciers_cci project (4000109873/14/I-NB). Upkeep and continuation of the inventory series is supported by PROMICE, funded by the Geological Survey of Denmark and Greenland (GEUS) and the Danish Ministry of Climate, Energy and Utilities under the Danish Cooperation for Environment in the Arctic (DANCEA), conducted in collaboration with DTU Space (Technical University of Denmark) and Asiaq Greenland Survey. (2024-10-03)",
```
Comment (PennyHow) [inline products/esa-griml-iml-fv1/collection.json:5]: The date in brackets at the end of this description does not need to be there. It is automatically generated by GEUS Dataverse. But I can understand keeping it if you would like this to follow exactly the description on the GEUS Dataverse.

File: products/esa-griml-iml-fv1/collection.json [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+        "title": "Open Science Catalog"
+      },
+      {
+        "rel": "via",
+        "href": "https://doi.org/10.22008/FK2/MBKW9N",
+        "title": "Access"
+      },
+      {
+        "rel": "via",
+        "href": "https://doi.org/10.1038/s41598-021-83509-1",
```
Comment (PennyHow) [inline products/esa-griml-iml-fv1/collection.json:20]: This is a DOI reference to the 2021 paper presenting the previous Greenland inventory dataset. Can we update this once we have our new dataset paper published?

File: products/esa-griml-iml-fv1/collection.json [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+        "type": "application/json",
+        "title": "EO Mission: Landsat-9"
+      },
+      {
+        "rel": "related",
+        "href": "../../eo-missions/sentinel-1/catalog.json",
+        "type": "application/json",
+        "title": "EO Mission: Sentinel-1"
+      },
+      {
```
Comment (PennyHow) [inline products/esa-griml-iml-fv1/collection.json:65]: Suggestion: add Sentinel-2 reference here also, as Sentinel-2 is another key dataset used.

```suggestion
     {
        "rel": "related",
        "href": "../../eo-missions/sentinel-2/catalog.json",
        "type": "application/json",
        "title": "EO Mission: Sentinel-2"
      },

```

File: products/esa-griml-iml-fv1/collection.json [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+      "cryosphere"
+    ],
+    "osc:variables": [
+      "land-surface-temperature"
+    ],
+    "osc:missions": [
+      "landsat-8",
+      "landsat-9",
+      "sentinel-1"
+    ],
```
Comment (PennyHow) [inline products/esa-griml-iml-fv1/collection.json:92]: Add Sentinel-2 here also.

```suggestion
    "sentinel-2"
    ],

```

File: products/esa-griml-iml-fv1/collection.json [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+            "2016-01-01T00:00:00Z",
+            "2023-01-01T23:59:59Z"
+          ]
+        ]
+      }
+    },
+    "license": "proprietary",
+    "keywords": [
+      "Ice Marginal Lake",
+      "Land Surface Temperature"
```
Comment (PennyHow) [inline products/esa-griml-iml-fv1/collection.json:118]:
```suggestion
      "Lake Surface Temperature"
```

**Review by PennyHow — CHANGES_REQUESTED** [review]

**Review by edobrowolska — APPROVED** [review]

**Pull request merged**